ckanapi/cli/main.py

```diff
@@ -4,6 +4,7 @@
   ckanapi action ACTION_NAME
           [(KEY=STRING | KEY:JSON | KEY@FILE ) ... | -i | -I JSON_INPUT]
           [-j | -J] [[-c CONFIG] [-u USER] | -r SITE_URL [-a APIKEY] [-g]]
+          [--insecure]
```

Comment:
please add this to the other subcommands in the docstring. docopt won't allow it to be passed if it's not present in each one.

Reply:
np, thanks for catching that. just added.